src/utils/storage/index.ts

```diff
@@ -87,6 +94,13 @@ const StorageUtils = {
   lastSelectedMenu: storageBuilder(StorageKeysEnum.LastSelectedMenu),
   blockingStatus: storageBuilder(StorageKeysEnum.BlockingStatus),
   blockedStatus: storageBuilder(StorageKeysEnum.BlockedStatus),
+  fetchInitialData: storageBuilder(StorageKeysEnum.FetchInitialData),
+  channelSignedTimeStamps: storageBuilder(StorageKeysEnum.ChannelSignedTimeStamps),
+  channelAnonTimeStamps: storageBuilder(StorageKeysEnum.ChannelAnonTimeStamps),
+  channelDetailTimeStamps: storageBuilder(StorageKeysEnum.ChannelDetailTimeStamps),
+  feedChatTimeStamps: storageBuilder(StorageKeysEnum.FeedChatTimeStamps),
+  anonymousNotificationTimeStamp: storageBuilder(StorageKeysEnum.AnonymousNotificationTimeStamps),
+  signedNotificationTimeStamp: storageBuilder(StorageKeysEnum.SignedNotificationTimeStamp),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the key name for anonymous notifications timestamp. It should match the key defined in `StorageKeysEnum`.
```diff
-  anonymousNotificationTimeStamp: storageBuilder(StorageKeysEnum.AnonymousNotificationTimeStamps),
+  anonymousNotificationTimeStamp: storageBuilder(StorageKeysEnum.AnonymousNotificationTimeStamp),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->